src/view/templates/main/popup-comments.js

```diff
@@ -0,0 +1,68 @@
+import dayjs from 'dayjs';
+
+const template = (data) => {
+  const p = data;
+  const comments = p.commentsData;
```

Comment:
Чтобы не делать несколько присваиваний можно пользоваться деструктуризацией
```suggestion
  const {commentsData:comments} = data;
```